backends/ref/ceed-ref-operator.c

```diff
@@ -18,93 +18,392 @@
 #include <string.h>
 #include "ceed-ref.h"
 
+CeedElemRestriction CEED_RESTRICTION_IDENTITY = NULL;
```

Comment:
I think this is being seen by other backends when they are built, which is not my intention.